XCUITests/DataManagementTests.swift

```diff
@@ -30,9 +30,12 @@ class DataManagementTests: BaseTestCase {
         waitforExistence(app.tables["Search results"])
         XCTAssertEqual(app.tables["Search results"].cells.count, 2)
         navigator.performAction(Action.TapOnFilterWebsites)
-        XCUIApplication().buttons["Cancel"].tap()
+        app.typeText("foo")
         XCTAssertEqual(app.tables["Search results"].cells.count, 0)
-        XCUIApplication().tables/*@START_MENU_TOKEN@*/.staticTexts["Show More"]/*[[".cells[\"ShowMoreWebsiteData\"].staticTexts[\"Show More\"]",".staticTexts[\"Show More\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        app.buttons["Cancel"].tap()
+        app.tables/*@START_MENU_TOKEN@*/.staticTexts["Show More"]/*[[".cells[\"ShowMoreWebsiteData\"].staticTexts[\"Show More\"]",".staticTexts[\"Show More\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
         XCTAssertNotEqual(app.tables.cells.count, 9)
+        navigator.performAction(Action.AcceptClearAllWebsiteData)
+        XCTAssertEqual(app.tables.cells.count, 1)
```

Comment:
It will be required to add a waiter here, I got intermittent failures after i did the commit :(.

Reply:
Okay, you can add a waitforExistence there